doc/Type/Block.pod6

```diff
@@ -10,11 +10,13 @@ A C<Block> is a code object meant for small-scale code reuse.  A block is
 created syntactically by a list of statements enclosed in curly braces.
 
 Without an explicit signature or placeholder arguments, a block has C<$_>
-as a positional argument
+as a positional argument. Which defaults to the outer scope's $_. Thus it
+will inherit the L<topic|/language/variables#index-entry-topic_variable> if possible.
 
     my $block = { uc $_; };
     say $block.^name;           # OUTPUT: «Block␤»
     say $block('hello');        # OUTPUT: «HELLO␤»
+    say {;}.signature;          # OUTPUT: «(;; $_? is raw = OUTER::<$_>)␤»
 
```

Comment:
Maybe add some example here?
```perl6
$_ = "hello";
my &block = { uc $_; };
say block # HELLO
```

You probably need to use the & form so that you can invoke it without any argument.